src/index.ts

```diff
@@ -10,11 +10,17 @@ import * as ClientStorage from './storage';
 
 //this is necessary to export the type definitions used in this file
 import './global';
-import { validateCrypto } from './utils';
+import { validateCrypto, getUniqueScopes } from './utils';
 
 export default async function createAuth0Client(options: Auth0ClientOptions) {
   validateCrypto();
 
+  if (options.useRefreshTokens) {
+    options.scope = options.scope
+      ? getUniqueScopes(options.scope, 'offline_access')
```

Comment:
Could this util just handle undefined?

Reply:
Yes, in fact it already does. I've simplified its usage here as a result and updated some tests.